icu4c/source/tools/icuexportdata/icuexportdata.cpp

```diff
@@ -164,6 +164,7 @@ void dumpBinaryProperty(UProperty uproperty, FILE* f) {
     fputs("[[binary_property]]\n", f);
     fprintf(f, "long_name = \"%s\"\n", fullPropName);
     if (shortPropName) fprintf(f, "short_name = \"%s\"\n", shortPropName);
+    fprintf(f, "uproperty = %d\n", uproperty);
```

Comment:
i don't know if it should be called uproperty, but it seems to make sense

Reply:
other reasonable names would be `discriminant` or `discr` like we did for values

Reply:
Do you think either of those are better than uproperty? Fine to pick anything.

Reply:
If I had to pick one I'd pick `discriminant` to be more consistent with the values, and to emphasize what this value represents. `uproperty_discriminant` or `uproperty_discr` also work.

Reply:
I like uproperty_discr